src/commands/service/doc.ts

```diff
@@ -12,6 +13,7 @@ export default class ServiceDoc extends Command {
 
   static flags = {
     ...Command.flags,
+    save: flags.string({char: 's', description: 'Save to specified file or allow to overwrite the default readme file when used as empty or create a new one if not exists'}),
```

Comment:
hum in fact the ux is weird with `--save file` maybe it should be one of these:
- `--save --filename xxx`
- `--output file`

What do you think?

Reply:
1st opt is ok with me. I don't know why cmd lib requires a following `=` for empty values on flags.

Reply:
why not simply `--save` without asking the user to set the path to the file?
Let's keep it simple. If the guy want to output the content in another file, then he can copy / paste or use >>

Reply:
From the example, if the flag is of type bool, then user doesn't have to set a value or =
https://oclif.io/docs/flags

Reply:
agree, KISS

Reply:
done